packages/dropzone/stories/dropzone.stories.ts

```diff
@@ -106,24 +120,50 @@ Dragged.args = {
     isDragged: true,
 };
 
+export const Filled = (args: StoryArgs): TemplateResult => {
+    return html`
+        <sp-dropzone id="dropzone" ?filled=${args.isFilled}>
+            Filled dropzone
+        </sp-dropzone>
+    `;
+};
+Filled.args = {
+    isFilled: true,
+};
+
```

Comment:
Probably nitpicking but can we make sure that all our stores look same? i.e., like the other stories of dropzone i'd like it to follow the same design with illustrated message and whatever else. 
Also on the side since we try to (or atleast in theory we do) use the same template for all our stories to show different component states, it would be better if we can reuse the same template without rewriting the same code in stories. Not a blocker. Just wanted to get your opinion on making our codebase better.

Reply:
That is a great point, and I think we should really consider a holistic approach to this (maybe outside the scope of this PR). But let me elaborate: since we use Storybook for both component demos and VRT, we need to balance showcasing component functionality with isolating each story's features to test specific functionalities. For example, if we change the `sp-action-button` or the `sp-illustrated-message`, it will cause `sp-dropzone` screenshots to fail. Is this acceptable? Maybe 🤷  However, ideally, we should have an additional layer of component demos that are independent of VRT so Storybook could focus solely on component testing in isolation.
I will follow-up on this.

Reply:
This is a great discussion thread! We should definitely talk more on this. Currently the storybook config are tightly coupled to docs site demos and we are tightly coupled here! We are trying to create some obfuscated demos which will cater for Desktop and Mobile versions to be represented in docs site and are independent of storybook. 
[This](https://m3.material.io/components/dialogs/guidelines#7e98ae4b-9b0f-4fc0-93c8-e811efffb952) looks like a good and informative way to represent component behaviours: